.github/workflows/compile.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Build RPM
         run: make CONTAINER_RUNTIME="docker" test-rpmbuild
       - name: Upload crc-driver-libvirt artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
Would be good to also update the dependabot configuration so that it does this for us, see https://github.com/crc-org/vfkit/blob/aa3cf4c94b50834b5f569622e5daaded9e0c2ca3/.github/dependabot.yml#L13-L17 for an example of this.

Reply:
I see in one of the later commits that there's also a `tools` directory, we should also add it to dependabot config file.

Reply:
Done, both are added.